src/inject.ts

```diff
@@ -68,12 +68,10 @@ export function resolveInjections(
 function normalizeInject(
   raw: ComponentInjectOptions | undefined
 ): ObjectInjectOptions | undefined {
-  if (isArray(raw)) {
-    const res: ObjectInjectOptions = {}
-    for (let i = 0; i < raw.length; i++) {
-      res[raw[i]] = raw[i]
-    }
-    return res
-  }
-  return raw
+  return isArray(raw)
+    ? raw.reduce((res, item) => {
```

Comment:
I'd prefer to keep the for loop. it's more expressive in general faster than reduce.

Reply:
I've switched back to for, but with a little change, I think it's better to minimize the use of for in the if condition